composer.json

```diff
@@ -15,28 +15,28 @@
         }
     ],
     "require": {
-        "php": "^8.0",
+        "php": "^8.1",
         "symfony-cmf/routing": "^3.0",
-        "symfony/framework-bundle": "^6.0"
+        "symfony/framework-bundle": "^6.4 || ^7.0"
     },
     "require-dev": {
-        "jackalope/jackalope-doctrine-dbal": "^1.3",
-        "doctrine/phpcr-odm": "^1.4|^2.0",
-        "symfony/phpunit-bridge": "^6.0",
-        "matthiasnoback/symfony-dependency-injection-test": "^4.1.0",
-        "matthiasnoback/symfony-config-test": "^4.1.0",
-        "doctrine/orm": "^2.9",
-        "symfony-cmf/testing": "dev-master as 4.2.0",
         "doctrine/data-fixtures": "^1.0.0",
-        "symfony/form": "^6.0",
-        "symfony/translation": "^6.0",
-        "symfony/validator": "^6.0",
-        "symfony/security-bundle": "^6.0",
         "doctrine/doctrine-bundle": "^2.0",
-        "symfony/twig-bundle": "^6.0",
+        "doctrine/orm": "^2.9 || ^3.0",
+        "doctrine/phpcr-bundle": "^3.0",
+        "doctrine/phpcr-odm": "^2.0",
+        "jackalope/jackalope-doctrine-dbal": "^1.3 || ^2.0",
+        "matthiasnoback/symfony-dependency-injection-test": "^4.1.0 || ^5.1.0",
+        "matthiasnoback/symfony-config-test": "^4.1.0 || ^5.1.0",
+        "symfony/phpunit-bridge": "^7.0.3",
+        "symfony/form": "^6.0 || ^7.0",
         "symfony/monolog-bundle": "^3.5",
-        "doctrine/phpcr-bundle": "^2.3",
-        "symfony/serializer": "^6.0",
+        "symfony/security-bundle": "^6.0 || ^7.0",
+        "symfony/serializer": "^6.0 || ^7.0",
+        "symfony/translation": "^6.0 || ^7.0",
+        "symfony/validator": "^6.0 || ^7.0",
+        "symfony/twig-bundle": "^6.0 || ^7.0",
+        "symfony-cmf/testing": "dev-sf7 as 4.2.0",
```

Comment:
change to ^5.0.0 when https://github.com/symfony-cmf/Testing/pull/215 is merged and released